.github/workflows/ci_test.yml

```diff
@@ -76,6 +76,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: bash tools/run_build.sh test_editable_mode
+  test_python_code_only:
+    name: Fast build to run python-only tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+        with:
+          python-version: 3.5
+      - uses: actions/checkout@v2
+      - run: pip install -r tools/install_deps/tensorflow-cpu.txt -r tools/install_deps/pytest.txt
```

Comment:
Any speedup caching pip packages? https://github.com/actions/cache/blob/master/examples.md#simple-example

Reply:
I tried, but I removed it in https://github.com/tensorflow/addons/pull/1666/commits/00832394a4d6a948eb76b139b4cfe984c9d3f092 because we didn't get any speed benefits.

Reply:
Have you tried another build After the cache build?

Reply:
Yes, you can take a look for yourself since the build logs are public for each commit.